valhalla/sif/dynamiccost.h

```diff
@@ -888,6 +888,12 @@ class DynamicCost {
   bool penalize_uturns_;
 
   bool exclude_unpaved_{false};
+
+  // HOT/HOV flags
+  bool include_hot_lanes_{false};
+  bool include_hov2_lanes_{false};
+  bool include_hov3_lanes_{false};
```

Comment:
nit: seems odd to have the `lanes_` suffix here. someone just glacing at this might be confused if there is some semantic difference between these and the base options